tmc-langs-rust/src/main/java/fi/helsinki/cs/tmc/langs/rust/util/Constants.java

```diff
@@ -0,0 +1,10 @@
+
+package fi.helsinki.cs.tmc.langs.rust.util;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class Constants {
```

Comment:
You cannot make it package private because it's used outside of util. Also I prefer not adding qualifiers to names which are apparent from the package (Package + class name was supposed to be the unique identifier in Java).